public/video-ui/styles/components/_bar.scss

```diff
@@ -21,14 +21,31 @@
     }
   }
 
+  &__image img {
+      width: 160px;
+      height: auto;
+      margin-right: 15px;
+
+  }
+
+  &__image-placeholder {
+      background-color: $color800Grey;
+      width: 160px;
+      height: 90px;
+      text-align: center;
+      padding-top: 32px;
```

Comment:
It's to have the 'No Image' title in the centre of the blank image box. I'll upload an example screenshot